docs/dex-specification.md

```diff
@@ -665,14 +667,16 @@ The contract for the new staking pool is deployed using `StakingInitializable` c
 - reward per block (in reward tokens)
 - the start and end blocks
 - the pool limit per user (in staked tokens)
-- the multisig address
 - the limit of the number of blocks available for a user
+- the multisig address
 
 The function returns the address of a new staking pool contract.
 
 #### `StakingInitializable`
 
-The staking factory contract (`StakingFactory`) deploys the `StakingInitializable` contract when a new staking pool is deployed. This contract initializes a new pool.
+The staking factory contract (`StakingFactory`) deploys the `StakingInitializable` contract, which is a new staking pool. Staking factory initializes the pool after it is deployed with the `deployPool` function.
```

Comment:
...the pool after it is deployed in the `deployPool` function// or ...the pool after it is deployed during the `deployPool` function execution